frontend/front/src/pages/Public/Pages/SurveyPage.js

```diff
@@ -27,6 +28,8 @@ const STEP_THANKS = "PHASE_THANKS";
  * This page should handle all API calls so that component switching is easier to control
  */
 export const SurveyPage = () => {
+  const [validationStatus, setValidationStatus] = useState();
```

Comment:
Allows child component `CanonicalizationLoader` to communicate query result to its parent.